src/locales/en-US/commands/reddit.json

```diff
@@ -1,15 +1,23 @@
 {
 	"name": "reddit",
 	"description": "Get information from Reddit",
+	"subredditName": "subreddit",
+	"subredditDescription": "Get a random post from a specified subreddit.",
+	"postName": "post",
+	"postDescription": "Get the information of a specific post.",
 	"optionsRedditName": "reddit",
-	"optionsRedditDescription": "The subreddit to get a post from",
-	"invalidName": "Subreddits must be between 3 and 21 characters long, and contain only letters, numbers, or underscores. Additionally, the `/r/` and `r/` prefixes is also supported.",
+	"optionsRedditDescription": "The subreddit name to get a random post from. The `/r/` and `r/` prefixes are also supported.",
```

Comment:
Please revert the second sentence, it becomes long enough to be unreadable on phone and untranslatable in some languages (French and German most notably)